source/blog/plugin-sustainability-initiative.md

```diff
@@ -0,0 +1,126 @@
+
+## Insights from the community
+
+Initiation of this project involved [recruiting](https://napari.zulipchat.com/#narrow/channel/212875-general/topic/Announcing.20the.20plugin.20sustainability.20initiative!/with/544624119) community members from as many communities as possible and establishing the Plugin Sustainability Initiative. The working group [meets at different times](https://napari.org/stable/community/meeting_schedule.html) for global participation and includes members from diverse scientific (many different career levels and roles) and napari (ie. core team, plugin developers, and users) backgrounds. The most surprising part of the [working group outcomes](https://hackmd.io/@napari/plugin-wg) hasn't been the list of familiar challenges, but instead how much the community *wants to help build and shape solutions*.
+
+Plugin developers have volunteered to review each other's work. Instructors have offered to write documentation about Git and GitHub basics. Active developers have suggested programs where unmaintained plugins could find new maintainers. The coversation was never "what should napari do for us?" but rather "how can we work together to make this better?"
+
+In one of the working group meetings it was said well: plugin developers often see the core napari team as shephereds of the whole ecosystem, but at the same time, plugin developers rely on the core team in ways the core team doesn't always know about and can't always support. Ultimately, people want ways to be involved more broadly with the *whole ecosystem* without necessarily having to contribute to the core tools, we just need to send out the invitations and build the right infrastracture to make this possible.
+
+## What we're building, together
+
+The Plugin Sustainability Initiative has indentified three main objectives:
+
+### 1. Modernize plugin development infrastructure
+
+We're updating the [napari-plugin-template](http://github.com/napari/napari-plugin-template) and [documentation](https://napari.org/stable/plugins/index.html) to follow current best practices from resources including the venerable [PyOpenSci](https://www.pyopensci.org/),[Scientific Python](https://scientific-python.org/), and [Python Packaging User Guide](https://packaging.python.org/en/latest/). These new and updated resources will tailored to different experience levels, so that both beginners and experienced developers have personalized napari-specific guidance. This includes:
```

Comment:
```suggestion
We're updating the [napari-plugin-template](http://github.com/napari/napari-plugin-template) and [documentation](https://napari.org/stable/plugins/index.html) to follow current best practices from resources including the venerable [PyOpenSci](https://www.pyopensci.org/),[Scientific Python](https://scientific-python.org/), and [Python Packaging User Guide](https://packaging.python.org/en/latest/). These new and updated resources will be tailored to different experience levels, so that both beginners and experienced developers have personalized napari-specific guidance. This includes:
```